python/requirements_ml_docker.txt

```diff
@@ -4,14 +4,11 @@ ipython
 tblib
 
 # If you make changes to the torch versions, please also make the corresponding changes to `requirements_dl.txt`!
--f https://download.pytorch.org/whl/torch_stable.html
-torch==1.9.0+cu111
-torchvision==0.10.0+cu111
 
--f https://data.pyg.org/whl/torch-1.9.0+cu111.html
+--extra-index-url https://download.pytorch.org/whl/cu111  # for GPU versions of torch, torchvision
+--find-links https://data.pyg.org/whl/torch-1.9.0+cu111.html  # for GPU versions of torch-scatter, torch-sparse
+torch==1.9.0
+torchvision==0.10.0
```

Comment:
torch now provides custom index urls, eliminating the need for the plus-notation